push_notification/subscriber/services/subscriber_service.py

```diff
@@ -0,0 +1,48 @@
+from subscriber.models.subscriber import Subscriber
+
+
+class SubscriberService(object):
+
+    def __init__(self, subscriber, serializer):
+        self.subscriber = subscriber
+        self.serializer = serializer
```

Comment:
`SubscriberService` will need a subscriber model and subscriber Serializer for initialisation,
on implementation you will end up using/importing the subscriber model and subscriber Serializer  everywhere you need to use it. 
Instead i would recommend you can import the Serializer and model hare and use it directly.

Please feel free to share your thoughts on this.